api/src/Rbac/Item/Assignment.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+/*
+ * This file is part of the Spira framework.
+ *
+ * @link https://github.com/spira/spira
+ *
+ * For the full copyright and license information, please view the LICENSE file that was distributed with this source code.
+ */
+
+namespace Spira\Rbac\Item;
+
+/**
+ * Assignment represents an assignment of a role to a user.
+ */
+class Assignment
+{
+    /**
+     * @var string|int user ID
+     */
+    public $userId;
+    /**
+     * @return string the role name
+     */
+    public $roleName;
+    /**
+     * @var int UNIX timestamp representing the assignment creation time
```

Comment:
This isn't an int? It will be a timestamp string, or instance of carbon depending how it is processed by laravel.